core/arch/arm/mm/core_mmu.c

```diff
@@ -1622,7 +1622,7 @@ void core_mmu_map_region(struct mmu_partition *prtn, struct tee_mmap_region *mm)
 	vaddr_t vaddr = mm->va;
 	paddr_t paddr = mm->pa;
 	ssize_t size_left = mm->size;
-	int level;
+	unsigned int level;
```

Comment:
This should probably be a separate commit

Reply:
It is already in a separate commit, "core: core_mmu.[ch]: use U() for unsigned constants". Should I mention this change too in the commit message?

Reply:
If it is a direct consequence of using the `U()` constants then it's fine you can keep it like this and no need to mention it.

Reply:
Yes, there was a warning.